sponsors/forms.py

```diff
@@ -2,43 +2,48 @@
 from django.utils.text import slugify
 from django.utils.translation import ugettext_lazy as _
 
-from sponsors.models import SponsorshipBenefit, SponsorshipLevel, SponsorshipProgram
-
+from sponsors.models import SponsorshipBenefit, SponsorshipPackage, SponsorshipProgram
 
 
 class SponsorshiptBenefitsForm(forms.Form):
-    level = forms.ModelChoiceField(
-        queryset=SponsorshipLevel.objects.all(),
-        widget=forms.RadioSelect(), required=False,
+    package = forms.ModelChoiceField(
+        queryset=SponsorshipPackage.objects.all(),
+        widget=forms.RadioSelect(),
+        required=False,
     )
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        benefits_qs = SponsorshipBenefit.objects.select_related("program", "minimum_level")
+        benefits_qs = SponsorshipBenefit.objects.select_related("program")
         for program in SponsorshipProgram.objects.all():
             slug = slugify(program.name).replace("-", "_")
             self.fields[f"benefits_{slug}"] = forms.ModelMultipleChoiceField(
                 queryset=benefits_qs.filter(program=program),
-                widget=forms.CheckboxSelectMultiple(), required=False,
-                label=_(f"{program.name} Sponsorship Benefits")
+                widget=forms.CheckboxSelectMultiple(),
+                required=False,
+                label=_(f"{program.name} Sponsorship Benefits"),
             )
 
     @property
     def benefits_programs(self):
         return [f for f in self if f.name.startswith("benefits_")]
 
     @property
-    def benefits_by_levels(self):
-        levels_benefits = {}
-        for level in SponsorshipLevel.objects.all():
-            levels_benefits[level.id] = level.benefits.values_list("id", flat=True)
-        return levels_benefits
+    def benefits_by_package(self):
+        packages_benefits = {}
+        for package in SponsorshipPackage.objects.all():
+            packages_benefits[package.id] = package.benefits.values_list(
+                "id", flat=True
+            )
+        return packages_benefits
 
     def clean(self):
         cleaned_data = super().clean()
 
         if not any([cleaned_data.get(bp.name) for bp in self.benefits_programs]):
-            raise forms.ValidationError(_("You have to pick a minimum number of benefits."))
+            raise forms.ValidationError(
+                _("You have to pick a minimum number of benefits.")
+            )
```

Comment:
@ewdurbin are you using black to format the code?

Reply:
Yes, but it is not enforced in any way.

Reply:
I've generally been applying black formatting to each application that gets "major" work. So for this work I've been running `black sponsors/` before committing.

Reply:
I'm a +1 for using black. I'll run it before my next commits